pkgs/development/libraries/motif/default.nix

```diff
@@ -27,7 +27,7 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ libXp libXau ];
 
   prePatch = lib.optionalString (!demoSupport) ''
```

Comment:
Shouldn't this really be postPatch so that patches apply properly?

Reply:
I suppose so!

Reply:
Changed.